client/lib/signup/step-actions/index.js

```diff
@@ -377,8 +399,14 @@ export function setDesignOnSite( callback, { siteSlug, selectedDesign } ) {
 				body: { trim_content: true },
 			} )
 		)
-		.then( () => {
-			callback();
+		.then( () =>
+			wpcom.req.get( {
+				path: `/sites/${ siteSlug }/block-editor`,
+				apiNamespace: 'wpcom/v2',
+			} )
+		)
+		.then( ( data ) => {
+			callback( null, { isFSEActive: data?.is_fse_active ?? false } );
```

Comment:
Great! I wasn't sure how we were going to figure out whether the selected theme was an FSE theme. This works, but do you think the theme API should be modified so that it returns for each theme whether it's FSE? That way it'd work more like Gutenboarding, where the hardcoded list of available-designs has an is_fse flag.

Reply:
I don't think we need to modify the API, as we already have the `block-templates` tag, which indicates the theme supports the Site editor (it's used for both universal and block-only themes).

Reply:
Can't we modify the `sites/new` response to include the information that we fetch from `sites/%s/block-editor` for us, so we can avoid that extra request?

Reply:
The theme is actually being selected some time after the `sites/new` endpoint has been called. In the current incarnation of the flow the site is always created using the Zoologist theme, and then later changed after the user has seen the design picker.

You still make a valid point (we could modify `themes/mine` or `sites/%s/theme-setup` to return `is_fse_actve`) but just wanted to clarify in case there was confusion.

Reply:
@creativecoder any thoughts on the API response modification? ☝️